### PR TITLE
Add eip155 for Neon EVM networks

### PR DESCRIPTION
**Pull request by afalaleev**

### CODE REVIEW

File: _data/chains/eip155-245022926.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Neon EVM DevNet",
+  "chain": "Solana",
+  "network": "devnet",
+  "rpc": [
+    "http://proxy.devnet.neonlabs.org/solana"
```
Comment (vodolaz) [inline _data/chains/eip155-245022926.json]: httpS://proxy.devnet.neonlabs.org/solana

File: _data/chains/eip155-245022934.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Neon EVM MainNet",
+  "chain": "Solana",
+  "network": "mainnet",
+  "rpc": [
+    "http://proxy.mainnet.neonlabs.org/solana"
```
Comment (vodolaz) [inline _data/chains/eip155-245022934.json]: httpS://proxy.mainnet.neonlabs.org/solana

File: _data/chains/eip155-245022940.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Neon EVM TestNet",
+  "chain": "Solana",
+  "network": "testnet",
+  "rpc": [
+    "http://proxy.testnet.neonlabs.org/solana"
```
Comment (vodolaz) [inline _data/chains/eip155-245022940.json]: httpS://proxy.testnet.neonlabs.org/solana

**Review by s-medvedev — APPROVED** [review]

File: _data/chains/eip155-245022934.json (hunk 2 above)
Comment (afalaleev) [reply _data/chains/eip155-245022934.json]: Fixed

File: _data/chains/eip155-245022926.json (hunk 1 above)
Comment (afalaleev) [reply _data/chains/eip155-245022926.json]: Fixed

File: _data/chains/eip155-245022940.json (hunk 3 above)
Comment (afalaleev) [reply _data/chains/eip155-245022940.json]: Fixed

**Review by vodolaz — APPROVED** [review]

**Pull request merged**